webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/nodeconfig/NodeConfigurationCacheRepository.scala

```diff
@@ -165,7 +165,7 @@ object NodeConfigurationHash {
           } yield {
             NodeConfigurationHash(NodeId(nodeId), d, nodeInfoCache.toInt, paramCache.toInt, nodeContextCache.toInt, p.toSet)
           }
-        case _ => Left(s"Cannot parse node configuration hash: ${json.splitAt(20)}...")
+        case _ => Left(s"Cannot parse node configuration hash: ${json}...")
```

Comment:
You should make the whole message only printed in degug, it can be huge (and repeated in thousands of nodes), leading to log explosion